### PR TITLE
LG-11731 failure to proof url

### PR DESCRIPTION
**Pull request by Sgtpluck**
https://cm-jira.usa.gov/browse/LG-11731

This change adds the Failure to Proof URL view to the user experience documentation.

### CODE REVIEW

File: _data/errors.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ oidc:
         Check the `acr_values` parameter in your authentication request and ensure that you are passing either:
         - `http://idmanagement.gov/ns/assurance/ial/1` for Authentication Only applications, OR;
         - `http://idmanagement.gov/ns/assurance/ial/2` for Identity-Verification Permitted applications.
-        Learn more about IAL Service Level values here: [https://developers.login.gov/oidc/authorization/#service_level]https://developers.login.gov/oidc/authorization/#service_level){:target="_blank"}
+        Learn more about IAL Service Level values here: [https://developers.login.gov/oidc/authorization/#service_level](https://developers.login.gov/oidc/authorization/#service_level){:target="_blank"}
```
Comment (Sgtpluck) [inline _data/errors.yml:23]: this is just a little typo i noticed

File: _pages/user-experience/failure-proof.md [hunk 2]
```diff
@@ -13,12 +13,49 @@ sidenav:
...
+
+Want to discuss options for your Failure to Proof URL? <a
+  class="usa-link usa-link--external"
+  rel="noreferrer"
+  target="_blank"
+  href="https://zendesk.login.gov">
+  Contact us
+</a>
+
+[Next step: Check out our knowledge articles]({{ site.baseurl }}/user-experience/knowledge-articles/)
```
Comment (Sgtpluck) [inline _pages/user-experience/failure-proof.md:61]: there was no "next step" text in the content, so i made this up. suggestions always welcome!

**Review by Jeremy1026 — APPROVED** [review]

File: _pages/user-experience/failure-proof.md [hunk 3]
```diff
@@ -13,12 +13,49 @@ sidenav:
...
+- The Failure to Proof URL is available for users who are not able  to capture or upload photos of their ID.
+- Users see the Failure to Proof URL when they encounter errors verifying their ID, personal information, or phone number. Examples of this include:
+- Users who enter valid information, but are not able to pass Login.gov’s proofing process because we cannot verify their information.
+- Users who have run out of attempts to verify their information.
+
+<figure>
+  <img src="{{ site.baseurl }}/assets/img/missing-docs.png"
+       alt='A section of a Login.gov page with the heading "Are you missing one of these items?" and three links: "See a list of accepted state-issued IDs", "Learn more about verifying by phone or mail", and "Get help at Partner Agency"."'
+       class="display-block grid-col flex-auto flex-align-center margin-y-4">
+  <figcaption>An example of what users see when they do not meet the requirements to complete identity verification with Login.gov. The term “Partner Agency” will be replaced by what the agency configures for this portion of their integration.</figcaption>
```
Comment (nprimak) [inline _pages/user-experience/failure-proof.md:46]: Visual nitpick.. I'm noticing the caption text is center aligned but in Figma the text is aligned with the edges of the image

**Review by nprimak — CHANGES_REQUESTED** [review]

File: _pages/user-experience/failure-proof.md (hunk 3 above)
Comment (Sgtpluck) [reply _pages/user-experience/failure-proof.md:46]: hm that's true, but it also makes it visually inconsistent with an existing image here: https://developers.login.gov/overview/ 
i feel like it's preferable for our docs to be internally consistent? what do you think?

Comment (Sgtpluck) [reply _pages/user-experience/failure-proof.md]: anyway updated in https://github.com/18F/identity-dev-docs/pull/414/commits/ebb6fd7a14498743f42a997d0c81a54e576433d6 @nprimak

**Review by nprimak — APPROVED** [review]
LGTM!

File: _pages/user-experience/failure-proof.md [hunk 4]
```diff
@@ -13,12 +13,49 @@ sidenav:
...
+
+Want to discuss options for your Failure to Proof URL? <a
+  class="usa-link usa-link--external"
+  rel="noreferrer"
+  target="_blank"
+  href="https://zendesk.login.gov">
+  Contact us
+</a>
+
+[Next step: Create knowledge errors for Login.gov customer support]({{ site.baseurl }}/user-experience/knowledge-articles/)
```
Comment (karenstrother) [inline _pages/user-experience/failure-proof.md:61]: Change errors to articles

File: _pages/user-experience/failure-proof.md [hunk 5]
```diff
@@ -13,12 +13,49 @@ sidenav:
...
 
+### What is a Failure to Proof URL?
+Login.gov’s agency partners provide a link that their users are directed to if they are unable to verify their identity with Login.gov. This is called the Failure to Proof URL (e.g. `failure_to_proof_url`). This link routes users to a designated page on the agency partner’s site which is configured as part of the agency's integration.
+
+The Failure to Proof URL is shown to users throughout the identity proofing process. Some examples of when a user might encounter this are:
+
+- Users might navigate to this page if they don’t have a valid state-issued ID or Social Security number.
+- The Failure to Proof URL is available for users who are not able  to capture or upload photos of their ID.
+- Users see the Failure to Proof URL when they encounter errors verifying their ID, personal information, or phone number. Examples of this include:
+- Users who enter valid information, but are not able to pass Login.gov’s proofing process because we cannot verify their information.
```
Comment (karenstrother) [inline _pages/user-experience/failure-proof.md:39]: Should this bullet and the next be indented? The bullet above ends with a colon.

Comment (Sgtpluck) [reply _pages/user-experience/failure-proof.md:39]: ooh yes thanks

File: _pages/user-experience/failure-proof.md (hunk 4 above)
Comment (Sgtpluck) [reply _pages/user-experience/failure-proof.md]: if i must

File: _pages/user-experience/failure-proof.md (hunk 5 above)
Comment (Sgtpluck) [reply _pages/user-experience/failure-proof.md]: fixed in https://github.com/18F/identity-dev-docs/pull/414/commits/8a69618f0a0501dfe6bff39fd501bf240cadda5c

File: _data/errors.yml (hunk 1 above)
Comment (aduth) [inline _data/errors.yml:23]: Not directly relevant to the changes being made here, but why do we make this link open in a new window? (`target="_blank"`)

Related: https://www.w3.org/TR/WCAG20-TECHS/G200.html

**Pull request merged**